### ISSUE TITLE
V5: Add new item

### ISSUE THREAD

**Issue by PlayXman** [PlayXman/app-msa#6]
Add new item button ("plus button") shouldn't be in topbar but it should be as [main action button](https://material-ui.com/components/floating-action-button/#floating-action-button) on the page.

**Comment by PlayXman** [comment]
Moreover, it doesn't have to show those one line items. It could show regular items ui.

**Comment by PlayXman** [comment]
## TLDR
1. **"Add new item" button**: It will be not in topbar, but as [floating action button](https://material-ui.com/components/floating-action-button/#floating-action-button) in bottom right corner.
1. **Search modal**: It will be basicaly the same - showing from bottom with cross to close.
1. **Items**: They will use standard "item ui" instead of "list ui" they're using now. there will be only 2 actions - add (plus icon in action area of a card) and info (main click event on whole card; new tab with information).
1. **Search field**: Standard bordered text field with clear button. It should be the same width as item container (now it's wider).

**Comment by PlayXman** [comment]
done
